package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "description": "Project Platform Health",
   "scripts": {
-    "build": "neutrino build",
-    "start": "neutrino start",
-    "start:local": "BACKEND=http://localhost:3000 neutrino start",
-    "lint": "neutrino lint",
+    "build": "BACKEND=https://firefox-health-backend.herokuapp.com webpack --optimize-minimize",
```

Comment:
`--optimize-minimize` is a legacy option meant for webpack 3 and below. 
Use `--mode production` instead.

Reply:
I've filed webpack/webpack-cli#624 for making `--optimize-minimize` show a deprecation warning.